_posts/newsletter/next.md

```diff
@@ -10,17 +10,28 @@ categories: [Updates]
+
+We got to understand the excitement that this brings to our community by Tim Berners-Lee. The community is growing. New individuals are building a way for themselves, and we see their creations developing into some great applications. As we continue to grow, we need to keep up the momentum by supporting our builders and our community members. We saw a significant increase in activity this month due to recent press attention. Tim was recently featured in [New York Times](https://www.nytimes.com/2021/01/10/technology/tim-berners-lee-privacy-internet.html?smid=tw-share) and spoke at the [Reuters Next conference](https://www.reuters.com/article/us-tech-bernerslee-interview/father-of-the-web-tim-berners-lee-prepares-do-over-idUSKBN29H1JK). Like we said in [January’s Solid World](https://vimeo.com/498076263), our foundation is there. Now it’s time to develop.
+
+Ruben Verborgh shared a helpful resource he created for the Community Solid Server, its [recipes](https://github.com/solid/community-server-recipes) for interoperability. Osmar Olivo, VP of Product for Inrupt, let us know there is now Node.js support for Inrupt’s [Solid Client Libraries](https://docs.inrupt.com/developer-tools/javascript/client-libraries/). If you want to hear or see it in detail, you can watch the recording of Solid World [here](https://vimeo.com/solidworld/february2021).
+
+Three presenters gave us an idea of just what it’s like to create a Solid application. Our first presenter [Noel DeMartin](https://noeldemartin.com/) gave us a demo of [Media Kraken](https://noeldemartin.github.io/media-kraken/login). Noel showcased the different ways you can log into and store the media/entertainment data in the application and on your computer. One of the significant aspects of the presentation was when Noel opened the door for other applications to interoperate with Media Kraken. One thing that makes Solid unique is creating interoperability between all servers, clients, and applications. Noel’s presentation reminds us of how possible it is. He is also a great point of reference when starting. If you’re new, check out his [journal](https://noeldemartin.com/tasks/).
 
-### In Other News
+Next, Jasmine Leonard presented [PodExplorer](https://podexplorer.gigalixirapp.com/login). Jasmine is new to the Solid community, joining in late 2020. While learning how to get engaged she built her application. When using the app, you can access data from your pod, upload, delete and view various files. During the presentation, Jasmine shared a few resources that helped build the PodExplorer—mentioning places like the [Solid Forum](https://forum.solidproject.org/), [Solid Website](http://solidproject.org/), and [GitHub Repos](https://github.com/solid/solidproject.org). In the future, Jasmine expects to add additional functionality to PodExplore, such as: bulk actions, managing the access of data, and finding files in your pod. 
```

Comment:
```suggestion
Next, Jasmine Leonard presented [PodExplorer](https://podexplorer.gigalixirapp.com). Jasmine is new to the Solid community, joining in late 2020. While learning how to get engaged with Solid she built her application. When using the app, you can access data from your pod, upload, delete and view various files. During the presentation, Jasmine shared a few resources that helped build the PodExplorer—mentioning places like the [Solid Forum](https://forum.solidproject.org/), [Solid Website](http://solidproject.org/), and [GitHub Repos](https://github.com/solid/solidproject.org). In the future, Jasmine expects to add additional functionality to PodExplorer, such as: bulk actions, managing the access of data, and finding files in your pod. 
```

Reply:
"Learning how to get engaged" does feel a bit distant, even with the "with Solid" addition. How about "learning how to work with Solid"?